web/src/components/molecules/Asset/Asset/AssetBody/svgPreview.tsx

```diff
@@ -6,12 +6,19 @@ type Props = { url: string; svgRender: boolean };
 const SVGPreview: React.FC<Props> = ({ url, svgRender }) => {
   const [svgText, setSvgText] = useState("");
 
-  const fetchData = useCallback(() => {
-    fetch(url)
-      .then(res => res.text())
-      .then(val => {
-        setSvgText(val);
-      });
+  const fetchData = useCallback(async () => {
+    const res = await fetch(url, {
+      method: "GET",
+      headers: {
+        "Access-Control-Allow-Origin": "*",
+      },
+    });
+    if (res.status !== 200) {
+      setSvgText("Could not display svg");
```

Comment:
Sorry just remembered this needs to be i18n supported

Reply:
I will add i18n to all texts in Asset and AssetList pages in a new separate PR.

Reply:
Oh okay, I guess the i18n PR merged after you began the asset tasks. Gotcha